server/models/notifications/getNotifications.js

```diff
@@ -0,0 +1,39 @@
+const { pool } = require('../database')
+
+/**
+ *
+ * @param {*} param0
+ * @param {String} param0.username
+ * @param {Number} param0.limit
+ * @return {Promise}
+ *
+ */
+
+function getNotifications({ username, limit }) {
+  return new Promise((resolve, reject) => {
+    pool.query(
+      `SELECT notif.id id,heading,description,created_at,\`read\`,nmap.id map_key FROM notifications notif INNER JOIN (SELECT id,notification_id,\`read\` FROM user_notifications WHERE username=?) nmap ON nmap.notification_id=notif.id ORDER BY created_at DESC LIMIT ${limit}`,
```

Comment:
line 15: `notif.id id,`...syntax error?

Reply:
Nope, it would be just an alias for notif.id

Reply:
Means, when the user would get the result object, it would be 
```
{
"id" : .....
}
```
instead of
```
{
"notif.id" : .....
}
```